### PR TITLE
doc: update README & add QR code

### PR DESCRIPTION
**Pull request by msgui**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,16 @@ And here are links of other **HugeGraph** component/repositories:
 3. [hugegraph-commons](https://github.com/apache/incubator-hugegraph-commons) (**common & rpc** module)
 4. [hugegraph-website](https://github.com/apache/incubator-hugegraph-doc) (**doc & website** code)
 
+## Contact Us
+- Github Issues: Feedback on usage issues and functional requirements (priority)
+
+
+- Feedback Email: dev@hugegraph.apache.org
```
Comment (liuxiaocs7) [inline README.md:62]: Hi, @1216063060, you can refer this PR https://github.com/apache/incubator-hugegraph-toolchain/pull/462, pay attention to adding blank lines between title and content,  remove extra blank lines in contents.

Comment (1216063060) [reply README.md]: OK，3Q~

File: README.md [hunk 2]
```diff
@@ -55,6 +55,7 @@ And here are links of other **HugeGraph** component/repositories:
 3. [hugegraph-commons](https://github.com/apache/incubator-hugegraph-commons) (**common & rpc** module)
 4. [hugegraph-website](https://github.com/apache/incubator-hugegraph-doc) (**doc & website** code)
 
+
```
Comment (imbajin) [inline README.md:58]: remove extra line

File: README.md [hunk 3]
```diff
@@ -69,3 +70,12 @@ HugeGraph relies on the [TinkerPop](http://tinkerpop.apache.org) framework, we r
...
 
 You are welcome to contribute to HugeGraph, and we are looking forward to working with you to build an excellent open source community.
+
+## Contact Us
+
+- [Github Issues](https://github.com/apache/incubator-hugegraph/issues): Feedback on usage issues and functional requirements (priority)
+- Feedback Email: [dev@hugegraph.apache.org](dev@hugegraph.apache.org)
+- WeChat public account: Apache HugeGraph, welcome to scan this QR code to follow us.
+
+![new QR code](https://user-images.githubusercontent.com/87920097/242012297-306495b5-18be-4f58-9335-21ee3c201408.png)
```
Comment (imbajin) [inline README.md:80]: just use `QR-Code` as the name, and remember update the image url later

File: README.md [hunk 4]
```diff
@@ -69,3 +69,12 @@ HugeGraph relies on the [TinkerPop](http://tinkerpop.apache.org) framework, we r
...
 
 You are welcome to contribute to HugeGraph, and we are looking forward to working with you to build an excellent open source community.
+
+## Contact Us
+
+- [Github Issues](https://github.com/apache/incubator-hugegraph/issues): Feedback on usage issues and functional requirements (priority)
+- Feedback Email: [dev@hugegraph.apache.org](dev@hugegraph.apache.org)
+- WeChat public account: Apache HugeGraph, welcome to scan this QR code to follow us.
+
+<img src="https://github.com/apache/incubator-hugegraph-doc/blob/master/assets/images/wechat.png?raw=true" alt="QR png" width="350" />
```
Comment (imbajin) [inline README.md:79]: I use html tag to reduce the size of the image, other repo could refer this

**Review by imbajin — DISMISSED** [review]

**Review by liuxiaocs7 — DISMISSED** [review]

File: README.md (hunk 4 above)
Comment (msgui) [reply README.md:79]: Hi @imbajin ，Is this the picture you were expecting 
image

File: README.md [hunk 5]
```diff
@@ -69,3 +69,12 @@ HugeGraph relies on the [TinkerPop](http://tinkerpop.apache.org) framework, we r
...
 You are welcome to contribute to HugeGraph, and we are looking forward to working with you to build an excellent open source community.
+
+>Contact Us
+>------
+
+> - [Github Issues](https://github.com/apache/incubator-hugegraph/issues): Feedback on usage issues and functional requirements (priority)
+> - Feedback Email: [dev@hugegraph.apache.org](dev@hugegraph.apache.org)
+> - WeChat public account: Apache HugeGraph, welcome to scan this QR code to follow us.
+
+> <img src="https://github.com/apache/incubator-hugegraph-doc/blob/master/assets/images/wechat.png?raw=true" width="350" />
```
Comment (imbajin) [inline README.md:80]: why use the quota for this section？（a mistake？）

File: README.md [hunk 6]
```diff
@@ -69,3 +69,11 @@ HugeGraph relies on the [TinkerPop](http://tinkerpop.apache.org) framework, we r
 Thanks to TinkerPop, thanks to Titan, thanks to DataStax. Thanks to all other organizations or authors who contributed to the project.
 
 You are welcome to contribute to HugeGraph, and we are looking forward to working with you to build an excellent open source community.
+
+Contact Us
+---
+ - [GitHub Issues](https://github.com/apache/incubator-hugegraph/issues): Feedback on usage issues and functional requirements (priority)
```
Comment (imbajin) [inline README.md:75]: usr `high priority` instead of `priority`

File: README.md [hunk 7]
```diff
@@ -69,3 +69,11 @@ HugeGraph relies on the [TinkerPop](http://tinkerpop.apache.org) framework, we r
 Thanks to TinkerPop, thanks to Titan, thanks to DataStax. Thanks to all other organizations or authors who contributed to the project.
 
 You are welcome to contribute to HugeGraph, and we are looking forward to working with you to build an excellent open source community.
+
+Contact Us
+---
```
Comment (imbajin) [inline README.md:74]: add a space after the separate line

File: README.md (hunk 6 above)
Comment (imbajin) [inline README.md:75]:
```suggestion
 - [GitHub Issues](https://github.com/apache/incubator-hugegraph/issues): Feedback on usage issues and functional requirements (quick response)
```

File: README.md (hunk 7 above)
Comment (imbajin) [inline README.md:74]:
```suggestion
### Contact Us

---

```

File: README.md [hunk 8]
```diff
@@ -69,3 +69,11 @@ HugeGraph relies on the [TinkerPop](http://tinkerpop.apache.org) framework, we r
...
+ - [GitHub Issues](https://github.com/apache/incubator-hugegraph/issues): Feedback on usage issues and functional requirements (priority)
+ - Feedback Email: [dev@hugegraph.apache.org](mailto:dev@hugegraph.apache.org)
```
Comment (imbajin) [inline README.md:76]:
```suggestion
 - Feedback Email: [dev@hugegraph.apache.org](mailto:dev@hugegraph.apache.org) ([subscriber](https://hugegraph.apache.org/docs/contribution-guidelines/subscribe/) only)
```

notice people we need subscribe the dev mailing-list first

**Review by imbajin — APPROVED** [review]

**Review by JackyYangPassion — APPROVED** [review]

**Pull request merged**